frontend/src/UserRegistration.tsx

```diff
@@ -32,15 +33,18 @@ export default function UserRegistration() {
             }
          })
             .then(response => {
-               if (response.status >= 200 && response.status < 300) {
-                  return response.text();
+               setStatusCode(response.status);
```

Comment:
I wouldn't do that, because that triggers an unnecessary rerender of the component. I added some code in slack, that might solve that.